src/docs/development/packages-and-plugins/developing-packages.md

```diff
@@ -223,6 +223,11 @@ environment:
 To create a plugin package, use the `--template=plugin`
 flag with `flutter create`.
 
+Starting from flutter 1.20.0:
+Use the `--platforms` option to specify all the platforms that the plugin
```

Comment:
=> Use the `--platforms=` option followed by a comma separated list to specify the platforms that the plugin supports.